pages/login.page.js

```diff
@@ -0,0 +1,38 @@
+import BasePage from "./base.page";
+const PAGE_TIMEOUT = 10000;
+const BASE_PAGE = BasePage;
```

Comment:
This is not needed if you apply inheritance

Reply:
Are you referring to the constant declaration or to the import?

Reply:
The constant, yeah. The import is needed.